### ISSUE TITLE
Multiple topic consumers not working in the same Camel context

### ISSUE THREAD

**Issue by laurikimmel** [mpilone/hazelcastmq#9]
Using 
* HazelcastMQ 1.3
* Hazelcast 3.4 
* Camel 2.14.1
* Spring 4.1.4
 
Setting up multiple consumers on same HazelcastMQ topic in same Camel context will prevent  context to start. The exception message is 'Multiple consumers for the same endpoint is not allowed'.

More details:
```java
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'camelContext' defined in class org.test.CamelContextConfig: Invocation of init method failed; nested exception is org.apache.camel.FailedToStartRouteException: Failed to start route route2 because of Multiple consumers for the same endpoint is not allowed: Endpoint[hazelcastmq://topic:test]
```

Simple setup for the failure case would look something like
```java
    new RouteBuilder() {
        @Override
        public void configure() throws Exception {
            from("hazelcastmq:topic:test").to('log:consumer1');
            from("hazelcastmq:topic:test").to('log:consumer2');
        }
```
There is workaround. Camel context will start successfully when endpoint URIs are different. For example
```java
    new RouteBuilder() {
        @Override
        public void configure() throws Exception {
            from("hazelcastmq:topic:test?requestTimeout=20001").to('log:consumer1');
            from("hazelcastmq:topic:test?requestTimeout=20002").to('log:consumer2');
        }
```
Messages are passed to both consumers as expected.

**Comment by mpilone** [comment]
Thanks for the detailed report. I'll take a look at it.

**Comment by mpilone** [comment]
This is fixed in 1.4.0. The HzMq Camel endpoint was marked as supporting multiple consumers.

**Comment by laurikimmel** [comment]
Fixed. Good work!
